src/main/java/uk/gov/hmcts/ccd/data/casedataaccesscontrol/RoleAssignmentAttributesResource.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.hmcts.ccd.data.casedataaccesscontrol;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.io.Serializable;
+
+@Builder
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class RoleAssignmentAttributesResource implements Serializable {
+    private static final long serialVersionUID = -7106266789404292869L;
```

Comment:
do we need to define serialVersionUID

Reply:
Yes, we should generate one if we implement Serializable. Otherwise the default one will get computed, that might be or might not be good enough.